### PR TITLE
KTOR-3650 Fixed `appendIfNameAbsent` and `appendIfNameAndValueAbsent`

### PR DESCRIPTION
**Pull request by rsinukov**

### CODE REVIEW

File: ktor-utils/common/src/io/ktor/util/StringValues.kt [hunk 1]
```diff
@@ -395,15 +395,15 @@ public fun StringValuesBuilder.appendAll(builder: StringValuesBuilder): StringVa
  * Appends [name] [value] pair if there are no values associated with [name]
  */
 public fun StringValuesBuilder.appendIfNameAbsent(name: String, value: String): StringValuesBuilder = apply {
-    if (!contains(name)) return@apply
+    if (contains(name)) return@apply
```
Comment (Stexxe) [inline ktor-utils/common/src/io/ktor/util/StringValues.kt:398]: Could you please just get rid of `return` since there is only one if statement?

**Comment by e5l** [comment]
Hey, could you check failing test?

**Review by e5l — CHANGES_REQUESTED** [review]
please check the CI build

**Review by e5l — APPROVED** [review]
lgtm

**Pull request merged**